Configuration/DataProcessing/python/Impl/ppEra_Run3_pp_on_PbPb_approxSiStripClusters_2023.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+"""
+_ppEra_Run3_pp_on_PbPb_approxSiStripClusters_2023_
+
+Scenario supporting proton collisions
```

Comment:
Here too

Reply:
addressed in [ee44f85](https://github.com/cms-sw/cmssw/pull/42493/commits/ee44f8593ee9ec1424f8e8fe270c00f9a5dc91df)